packages/compass-connections/src/index.tsx

```diff
@@ -79,6 +79,10 @@ const ConnectionsComponent: React.FunctionComponent<{
   });
   const activeConnectionsInfo = useMemo(() => {
     return activeConnections.map((connection) => {
+      // TODO: we need the connection's DevtoolsConnectOptions so that the
```

Comment:
This is going to require some work to avoid the circular dependency problem between compass-connections and compass-assistant. Or to get access to the dataService for the connection from here or elsewhere so we can store (or have) the relevant DevtoolsConnectOptions available next to the connection id and connection string. So that the assistant can connect to the same thing compass is connected to.